host-dtc-overlay Not Found

When compiling a CHIP (pro, in this case) `rootfs` using this utility, the compile fails when attempting to locate a tar ball containing `host-dtc-overlay` which used to be hosted at @atenart 's repo:

```
>>> host-dtc-overlay 61bbb7e7719959dc70917ae855398d278afa99c7 Downloading
--2017-06-28 03:05:50--  https://github.com/atenart/dtc/archive/61bbb7e7719959dc70917ae855398d278afa99c7/dtc-overlay-61bbb7e7719959dc70917ae855398d278afa99c7.tar.gz
Resolving github.com (github.com)... 192.30.253.113, 192.30.253.112
Connecting to github.com (github.com)|192.30.253.113|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-06-28 03:05:50 ERROR 404: Not Found.

--2017-06-28 03:05:50--  http://sources.buildroot.net//dtc-overlay-61bbb7e7719959dc70917ae855398d278afa99c7.tar.gz
Resolving sources.buildroot.net (sources.buildroot.net)... 176.9.16.109
Connecting to sources.buildroot.net (sources.buildroot.net)|176.9.16.109|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-06-28 03:05:51 ERROR 404: Not Found.

package/pkg-generic.mk:73: recipe for target '/home/vagrant/CHIP-buildroot/output/build/host-dtc-overlay-61bbb7e7719959dc70917ae855398d278afa99c7/.stamp_downloaded' failed
make: *** [/home/vagrant/CHIP-buildroot/output/build/host-dtc-overlay-61bbb7e7719959dc70917ae855398d278afa99c7/.stamp_downloaded] Error 1
```

Indeed, it seems that the repo https://github.com/atenart/dtc no longer exists.  Is this tarball hosted elsewhere?  Can it be made available?  This completely breaks the `buildroot` process, which has so far been working very well.

There is a recent NTC forum post about this here from other users:  https://bbs.nextthing.co/t/issue-with-buildroot-make-no-https-github-com-atenart-dtc/17466

--- a/package/dtc-overlay/Config.in.host
+++ b/package/dtc-overlay/Config.in.host
@@ -6,4 +6,4 @@ config BR2_PACKAGE_HOST_DTC_OVERLAY
 
 	  This version is patched to support device tree overlays.
 
-	  https://github.com/atenart/dtc
+	  https://github.com/NextThingCo/dtc

--- a/package/dtc-overlay/dtc-overlay.mk
+++ b/package/dtc-overlay/dtc-overlay.mk
@@ -4,8 +4,8 @@
 #
 ################################################################################
 
-DTC_OVERLAY_VERSION = 61bbb7e7719959dc70917ae855398d278afa99c7
-DTC_OVERLAY_SITE = $(call github,atenart,dtc,$(DTC_OVERLAY_VERSION))
+DTC_OVERLAY_VERSION = 93f8c3a7d6bb37cc414a7116264361cb737f2966
+DTC_OVERLAY_SITE = $(call github,NextThingCo,dtc,$(DTC_OVERLAY_VERSION))
 DTC_OVERLAY_LICENSE = GPLv2+/BSD-2c
 DTC_OVERLAY_LICENSE_FILES = README.license GPL
 DTC_OVERLAY_DEPENDENCIES = host-bison host-flex

--- a/package/dtc-overlay/dtc-overlay.mk
+++ b/package/dtc-overlay/dtc-overlay.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-DTC_OVERLAY_VERSION = 93f8c3a7d6bb37cc414a7116264361cb737f2966
+DTC_OVERLAY_VERSION = 61bbb7e7719959dc70917ae855398d278afa99c7
 DTC_OVERLAY_SITE = $(call github,NextThingCo,dtc,$(DTC_OVERLAY_VERSION))
 DTC_OVERLAY_LICENSE = GPLv2+/BSD-2c
 DTC_OVERLAY_LICENSE_FILES = README.license GPL

